src/ServiceControl.Audit.AcceptanceTests.RavenDB/ServiceControl.Audit.AcceptanceTests.RavenDB.csproj

```diff
@@ -23,6 +24,7 @@
 
   <ItemGroup>
     <Compile Include="..\ServiceControl.Audit.AcceptanceTests\**\*.cs" Exclude="..\ServiceControl.Audit.AcceptanceTests\AcceptanceTestStorageConfiguration.cs;..\ServiceControl.Audit.AcceptanceTests\obj\**\*.*;..\ServiceControl.Audit.AcceptanceTests\bin\**\*.*" />
+    <Compile Remove="..\ServiceControl.Audit.AcceptanceTests\TestsFilter.cs" />
```

Comment:
Seems like some rearranging could make this unnecessary but :shrug:

Reply:
Yeah, I could put all those in the Exclude of the previous line, I feel it's more explicit/visible this way.